### PR TITLE
Add the first draft of release instructions

### PR DESCRIPTION
**Pull request by alirezazolanvari**
Add release instructions!

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri @alirezazolanvari Review 💯

File: Otherfiles/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		1. Add a new section
+		2. Add changelogs
+	3. Update all version tags
+		1. `index.html`
+	4. Update size of files
+		1. `index.html`
+	5. Update document
+		1. Remove `doc` folder
+		2. Copy/Paste `doc_html` folder (Step **11**)
+		3. Rename `doc_html` folder to `doc`
```
Comment (sadrasabouri) [inline Otherfiles/RELEASE.md:56]: This section is not clear.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thanks for your efforts guys.
@sepandhaghighi I think the `doc_html` and `doc` related tasks are not clear.

File: Otherfiles/RELEASE.md (hunk 1 above)
Comment (sadrasabouri) [reply Otherfiles/RELEASE.md:56]: That's a great solution.

Comment (alirezazolanvari) [reply Otherfiles/RELEASE.md:56]: > The folder named `doc_html` is the result of running the `doc_2_html.bat` script. We have the option to modify the script `doc_2_html.bat`, and give it the new name of `doc`. What is your suggestion?

That's also nice to me.

Comment (sepandhaghighi) [reply Otherfiles/RELEASE.md]: fixed in https://github.com/sepandhaghighi/pycm/pull/515/commits/ac40898aac6debfad1222d8cd861cb10a13b3e52

**Review by sadrasabouri — APPROVED** [review]

**Comment by alirezazolanvari** [comment]
@sepandhaghighi Thanks! Everything looks fine to me.

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**